.github/workflows/deploy.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Docs Deploy
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
Do you think it's worst specifying exact os version, ex [ubuntu-18.04](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on)

Reply:
I don't see a reason why we wouldn't want to always run on latest

Reply:
Unless some new versions of ubuntu can break something for us we should not worry. I still prefer using latest LTS however there should be no impact for us, I believe. Just wanted to make sure, thanks